src/kv/MemDB.cc

```diff
@@ -1,653 +0,0 @@
-// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
i guess the idea of MemDB is used as a stub for understanding the performance impact on the i/o path of using a KV store backend like RocksDB. so we can short-circuit this part when we want to analyze the performance characteristics of the whole system assuming rocksdb is not the bottleneck.

Reply:
Yup, that's my understanding too. However, is `MemDB` actively used?